### ISSUE TITLE
Clauses aren't allowed be tuples

### ISSUE THREAD

**Issue by mbargull** [pysathq/pysat#3]
The `.add_clause` methods require a clause to be a `list` of integers. It would be nice if not only `list`s but also other iterables, most notably `tuple`s, were valid inputs.
As a reference, the `pycosat` interface for PicoSAT implements this at https://github.com/ContinuumIO/pycosat/blob/0.6.3/pycosat.c#L91-L145.

**Comment by alexeyignatiev** [comment]
Thank you for the suggestion! It does make sense to support other types of Python iterables and I will try to implement something similar this in the nearest future. I will let you know once it is done.

**Comment by alexeyignatiev** [comment]
Hi @mbargull. I found some time to implement this feature today. Please, check version 0.1.3.dev19 of the toolkit (either install it from pip or pull the latest commits from the repo and compile manually). It should work now. Please, confirm if everything works as expected on your machine, in which case, please, close the issue.

Also, do not hesitate to open another issue if you spot any other problem! Thanks again.

**Comment by mbargull** [comment]
Works like a charm, thanks!
